app/controllers/help_controller.rb

```diff
@@ -1,4 +1,8 @@
 class HelpController < ApplicationController
-	before_filter -> { @css_framework = :bootstrap }
-	layout ->(c) { request.format == :mobile ? "application" : "with_header_with_footer" }
-end
+  before_filter -> { @css_framework = :bootstrap }
+  layout -> (c) { request.format == :mobile ? "application" : "with_header_with_footer" }
```

Comment:
Unused block argument - `c`. If it's necessary, use `_` or `_c` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.<br>Line is too long. [89/80]